### ISSUE TITLE
write_uvh5 errors with numpy version 1.19

### ISSUE THREAD

**Issue by PyxieLouStar** [RadioAstronomySoftwareGroup/pyuvdata#868]
When calling write_uvh5 using numpy version 1.19, I get the following error: 
```
Traceback (most recent call last):
  File "aws_van_vleck.py", line 63, in <module>
    uv.write_uvh5(path + obs + '_mini_vv_corr_220620.uvh5')
  File "/home/ubuntu/.local/lib/python3.6/site-packages/pyuvdata/uvdata/uvdata.py", line 7531, in write_uvh5
    data_write_dtype=data_write_dtype,
  File "/home/ubuntu/.local/lib/python3.6/site-packages/pyuvdata/uvdata/uvh5.py", line 1192, in write_uvh5
    self._write_header(header)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/pyuvdata/uvdata/uvh5.py", line 1049, in _write_header
    header["antenna_names"] = np.string_(self.antenna_names, keepdims=True)
TypeError: 'keepdims' is an invalid keyword argument for this function
```
This is remedied by downgrading numpy to 1.18.

**Comment by bhazelton** [comment]
@plaplant I wanted to make sure you'd seen this.

**Comment by plaplant** [comment]
Thanks for pointing this out! I looked at [the v1.19.0 release notes](https://github.com/numpy/numpy/releases/tag/v1.19.0), and I didn't see this explicitly called out as a change. But they do mention a general deprecation of python2-compatibility shims, so it's probably part of that.

I'll make a PR to address this. I'm not sure why we had used this option to begin with, so hopefully it's an easy fix.
